docs/tutorials/vault/creating-vault-issuers.rst

```diff
@@ -54,6 +54,7 @@ We can now create a cluster issuer referencing this secret:
       vault:
         path: pki_int/sign/example-dot-com
         server: https://vault
+        caBundle: ...
```

Comment:
Instead of `...` can we have something like `<base64 encoded caBundle PEM file>`